.markdownlintignore

```diff
@@ -124,6 +127,10 @@ seed-users.json
 # Storybook
 **/storybook-static
 
+# Status
+/libs/status/crate/src/gen
+
+
```

Comment:
```suggestion
```

Reply:
This is removed in https://github.com/hashintel/hash/pull/1810